tensorflow/python/kernel_tests/decode_bmp_op_test.py

```diff
@@ -64,6 +67,16 @@ def testex1(self):
       decoded = decode.eval()
       self.assertAllEqual(decoded, img_bytes)
 
+  def testGrayscale(self):
+    prefix_path = "tensorflow/core/lib"
+    # Read a grayscale 512*512 bmp and verify shape
+    path = os.path.join(prefix_path, "bmp", "testdata", "lena512.bmp")
```

Comment:
We'd like to avoid binary files as much as possible. Since this is testing the decoder, not the reader, could you just use code to define test data (as done above).

Reply:
Thanks @martinwicke. The PR has been updated with binary file `lena512.bmp` removed.